src/openrct2/world/map.c

```diff
@@ -3545,6 +3545,8 @@ sint32 map_element_get_banner_index(rct_map_element *mapElement)
             ((mapElement->properties.scenerymultiple.colour[1] & 0xE0) >> 5);
     case MAP_ELEMENT_TYPE_WALL:
         sceneryEntry = get_wall_entry(mapElement->properties.wall.type);
+        if (sceneryEntry == NULL)
```

Comment:
Why not combine the conditions into `if (sceneryEntry == NULL || sceneryEntry->wall.scrolling_mode == 0xFF)`?

Reply:
I did that in other places, but missed that one

Reply:
done